vocab/en-us/GetKeyword.voc

```diff
@@ -0,0 +1,4 @@
+whats
+what is
+give me
+show me
```

Comment:
"Show" is an odd word here.  In the future (18.02) we will be using "show me" to indicate things that will be sent to an associated screen.  If you would intend this to show something, then I'm fine leaving this.

We should allow "tell me"